plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -97,60 +114,102 @@ func actionHandler(request *service.ActionRequest) error {
 	case addTfAction:
 		srcNamespace, err := request.Payload.String(srcNamespaceCol)
 		if err != nil {
-			log.Printf("Failed to get srcNamespace at string : %s", err)
+			log.Printf("Failed to get srcNamespace as string: %s", err)
+			return nil
 		}
 		if match := regExpMatch(namespaceStrPattern, srcNamespace); !match {
+			log.Printf("Failed to match namespace %s with k8s namespace pattern %s", srcNamespace, namespaceStrPattern)
```

Comment:
```suggestion
			log.Printf("Failed to match namespace %s with K8s Namespace pattern %s", srcNamespace, namespaceStrPattern)
```
so do the following logs

Reply:
Log changed.